engines/grim/lua_v1.cpp

```diff
@@ -497,8 +497,14 @@ void Lua_V1::GetSpeechMode() {
 }
 
 void Lua_V1::GetDiskFreeSpace() {
+	if(g_grim->getGameType() == GType_MONKEY4 &&
```

Comment:
We prefer a space between the if and the ( for consistency (and having it consistent helps with searching in the code base for instance).